src/Models/Custom.ts

```diff
@@ -0,0 +1,8 @@
+import { SimplifiedAlbumObject } from "./SpotifyObjects/AlbumObjects"
+import { ArtistObject } from "./SpotifyObjects/ArtistObjects"
+import { SimplifiedPlaylistObject } from "./SpotifyObjects/PlaylistObjects"
+import { TrackObject } from "./SpotifyObjects/TrackObjects"
+
+export type searchTypes = "artist" | "playlist" | "track" | "album"
```

Comment:
Nitpicky, but I like type names to have capitalized first letters: `SearchTypes`, `Items` (maybe make this `SearchItems` for clarity?), `ResponseItems` (maybe also add `Search` in front of this?)